### PR TITLE
[38037812] Fix RESTQ to have a default selection, better localization, and the new date/time control

### PR DESCRIPTION
**Pull request by emerencia**
The new date/time control is backwards compatible with the old one.

Fixes https://freedcamp.com/view/2354427/tasks/panel/task/38037812

### CODE REVIEW

File: projects/sport-data-valley/seeds/questionnaires/restq.rb [hunk 1]
```diff
@@ -77,9 +80,13 @@ def question(id, question_en, question_nl)
...
...-item"><p><label for="v7_zelden"><input name="content[v7]" type="radio" id="v7_zelden" value="Seldom" required="required" class="validate"><span>Seldom</span></label></p></div><div class="likert-item"><p><label for="v7_soms"><input name="content[v7]" type="radio" id="v7_soms" value="Sometimes" required="required" class="validate"><span>Sometimes</span></label></p></div><div class="likert-item"><p><label for="v7_regelmatig"><input name="content[v7]" type="radio" id="v7_regelmatig" value="Often" required="required" class="validate"><span>Often</span></label></p></div><div class="likert-item"><p><label for="v7_vaak"><input name="content[v7]" type="radio" id="v7_vaak" value="More often" required="required" class="validate"><span>More often</span></label></p></div><div class="likert-item"><p><label for="v7_heel_vaak"><input name="content[v7]" type="radio" id="v7_heel_vaak" value="Very often" required="required" class="validate" checked><span>Very often</span></label></p></div><div class="likert-item"><p><label for="v7_altijd"><input name="content[v7]" type="radio" id="v7_altijd" value="Always" required="required" class="validate"><span>Always</span></label></p></div></div></div></div>',
```
Comment (frbl) [inline projects/sport-data-valley/seeds/questionnaires/restq.rb:86]: this is nasty

File: projects/sport-data-valley/seeds/questionnaires/restq.rb [hunk 2]
```diff
@@ -40,31 +40,34 @@ def question(id, question_en, question_nl)
...
-  type: :date,
-  title: {en: 'Date', nl: 'Datum'}
-}, {
-  id: :v6,
-  type: :time,
-  title: {en: 'Time', nl: 'Tijd'},
-  hours_from: 0,
-  hours_to: 24
+  type: :date_and_time,
+  hours_id: :v6_uur,
```
Comment (frbl) [inline projects/sport-data-valley/seeds/questionnaires/restq.rb:65]: nice :-)

**Review by frbl — APPROVED** [review]
lgtm

File: projects/sport-data-valley/seeds/questionnaires/restq.rb (hunk 2 above)
Comment (emerencia) [reply projects/sport-data-valley/seeds/questionnaires/restq.rb:65]: Yep, it now uses the new date/time control and is full backwards compatible.

File: projects/sport-data-valley/seeds/questionnaires/restq.rb (hunk 1 above)
Comment (emerencia) [reply projects/sport-data-valley/seeds/questionnaires/restq.rb:86]: Yep.

**Pull request merged**